tabpy-tools/setup.py

```diff
@@ -1,13 +1,31 @@
-import versioneer
 try:
     from setuptools import setup
 except ImportError:
     from distutils.core import setup
 
+from pathlib import Path
+
+def read_version():
+    f = None
+    for path in ['VERSION', '../VERSION', '../../VERSION']:
```

Comment:
Does offering three VERSION files really do anything? Seems rather fragile to do this

Reply:
It is the same, but its location differs for when we run unit test, deploying package or running the app